src/interface/src/app/map/map-control-panel/map-control-panel.component.ts

```diff
@@ -170,20 +173,28 @@ export class MapControlPanelComponent implements OnInit {
           .map((pillar): ConditionsNode => {
             return {
               ...pillar,
-              filepath: pillar.filepath?.concat('_normalized'),
               layer: pillar.normalized_layer,
+              data_download_link: pillar.normalized_data_download_path ?
+	              BackendConstants.DOWNLOAD_END_POINT + '/' + pillar.normalized_data_download_path :
+                undefined,
               normalized: true,
               children: pillar.elements?.map((element): ConditionsNode => {
                 return {
                   ...element,
-                  filepath: element.filepath?.concat('_normalized'),
                   layer: element.normalized_layer,
+                  data_download_link: element.normalized_data_download_path ?
+                  BackendConstants.DOWNLOAD_END_POINT + '/' + element.normalized_data_download_path :
+                  undefined,
```

Comment:
any reason you left this undefined vs. just setting to data_download_link like the other datatypes?

Reply:
It actually had the same behavior since pillars and elements don't have a data_download_link value (only metric does). If the data download path isn't there, then it just doesn't have the download link in the info card.